docs/installing_deis/vagrant.rst

```diff
@@ -31,18 +31,20 @@ The ``Vagrantfile`` requires the plugin `vagrant-triggers`_. To install the plug
     website.
 
 
-Generate SSH Key
-----------------
-
-.. include:: ../_includes/_generate-ssh-key.rst
-
-
 Generate a New Discovery URL
 ----------------------------
 
 .. include:: ../_includes/_generate-discovery-url.rst
 
 
+SSH Key
```

Comment:
I still think this should be "Generate SSH Key" to be consistent with the other installation docs. Either that or add some actionable verb before "SSH Key" so people know what the paragraph is going to talk about -- "SSH Key" is too ambiguous.